_docs/tasks/security/plugin-ca-cert.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Plugging in root certificate, signing certificate and key
-overview: This task shows how operators can plug existing root certificate, signing certificate and key into Istio Citadel.
+overview: This task shows how operators can plug existing root certificate, signing certificate and key into Citadel.
```

Comment:
This task shows how operators can configure Citadel with existing root certificates, signing certificates and keys.

Reply:
Done.